src/components/AreaOfResponsibiltySelector.tsx

```diff
@@ -44,21 +44,11 @@ export const AreaOfResponsibilitySelector = () => {
   const organizationQuery = useQuery({
     enabled: areasOfResponsibilityIds.length > 0,
     queryKey: ['organizations', areasOfResponsibilityIds],
-    queryFn: async () => {
-      const areaPromises = areasOfResponsibilityIds.map(async (orgId) => {
-        return await fetchOrganization(orgId);
-      });
-      const organizations = await Promise.all(areaPromises);
-      const organizationOptions = buildOrganizationOptions(organizations);
-      if (!searchParams.get(TicketSearchParam.ViewingScope)) {
-        searchParams.set(TicketSearchParam.ViewingScope, organizationOptions.map((org) => org.id).join(','));
-        history.push({ search: searchParams.toString() });
-      }
-      return organizationOptions;
-    },
+    queryFn: async () => fetchOrganizations(areasOfResponsibilityIds),
     meta: { errorMessage: t('feedback.error.get_institution') },
   });
-  const organizationOptions = organizationQuery.data ?? [];
+  const organizations = organizationQuery.data ?? [];
+  const organizationOptions = buildOrganizationOptions(organizations);
```

Comment:
Ikke helt sikker her, men kan være det er like greit å la denne være i `queryFn` sånn at det gjøres bare en gang (når alle oranisasjoner hentes) i stedet for å gjøre det hver rerender.
Men da snakker vi sannsyligvis optimalisering som bruker ikke burde merke. Men kan være verdt å sjekke

Reply:
Gjorde det først, men da ble det kluss med queryen som brukes i default-value-wrapperen som brukte samme queryKey. Kunne jo bare hatt to forskjellige keys, men da cacher vi egentlig samme data to ganger. Wrapper-funksjonen trenger ikke organisasjonen som options-struktur (og det å dra den strukturen ut av komponenten høres ikke riktig ut heller). Så derfor valgte jeg å beholde samme queryKey, og la de to ulike komponentene behandle responsen på hver sin måte.